onnxruntime/python/tools/transformers/models/stable_diffusion/ort_utils.py

```diff
@@ -3,17 +3,23 @@
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
 
+import gc
+import logging
+import os
+import shutil
 from collections import OrderedDict
-from typing import Dict
+from typing import Any, Dict
 
 import torch
 
 import onnxruntime as ort
 from onnxruntime.transformers.io_binding_helper import TypeHelper
 
+logger = logging.getLogger(__name__)
+
 
 class OrtCudaSession:
```

Comment:
>OrtCudaSession [](http://example.com/codeflow?start=6&length=14)

Is it better to name it OrtNvidiaGPUSession?

Reply:
With slight modification, this class can run on AMD GPU.